src/main/java/org/skriptlang/skript/bukkit/itemcomponents/equippable/elements/CondEquipCompDamage.java

```diff
@@ -0,0 +1,60 @@
+package org.skriptlang.skript.bukkit.itemcomponents.equippable.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableExperiment;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableWrapper;
+
+@Name("Equippable Component - Will Lose Durability")
+@Description("Whether an item can be damaged when the wearer gets injured. "
+	+ "NOTE: Equippable component elements are experimental. Thus, they are subject to change and may not work as intended.")
+@Example("""
+	if {_item} will lose durability when hurt:
+		add "Damageable on injury" to lore of {_item}
+	""")
+@Example("""
+	set {_component} to the equippable component of {_item}
+	if {_component} won't lose durability on injury:
+		make {_component} lose durability when injured
+	""")
+@RequiredPlugins("Minecraft 1.21.2+")
+@Since("INSERT VERSION")
+public class CondEquipCompDamage extends PropertyCondition<EquippableWrapper> implements EquippableExperiment {
+
+	static {
+		Skript.registerCondition(CondEquipCompDamage.class, ConditionType.PROPERTY,
+			"%equippablecomponents% will lose durability (on [wearer['s]] injury|when [[the] wearer [is]] (hurt|injured))",
```

Comment:
Thoughts on `be damaged` as an alternative for `lose durability`? And `damaged` as another alternative for `hurt|injured`

Same goes for the Effect